templates/eks/amazon-eks-nodepool-cf.yaml

```diff
@@ -12,7 +12,7 @@ Metadata:
           - ClusterName
           - ClusterControlPlaneSecurityGroup
           - NodeSecurityGroup
-          - CustomSecurityGroup
+          - SecurityGroups
```

Comment:
Isn't it called CustomSecurityGroups?

Reply:
In the latest update (#3315) it was renamed to `SecurityGroups`, that's where the discrepancy was introduced which I'm trying to fix here.

Reply:
My2cents are to merge this and if we want to change the name again, let's do that in a different PR.